### PR TITLE
 cleanup Credit

### PR DESCRIPTION
**Pull request by Jahrkor**
Remove deprecated Credit parameters #6014

### CODE REVIEW

**Comment by cesium-concierge** [comment]
[Please sign the CLA before we review this PR.](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla)

Welcome to the Cesium community @Jahrkor!

Can you please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) so that we can review and merge this pull request?

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by pjcozzi** [comment]
We now have a CLA from @Jahrkor.

Thanks again for the contribution!

**Comment by hpinkos** [comment]
Thanks @Jahrkor!  You'll also need to remove the `deprecationWarning` and the `if` block from lines 36-40

**Comment by ggetz** [comment]
Hi @Jahrkor, thanks for the PR! 

It looks like you have some esLint failures, see [the build log](https://travis-ci.org/AnalyticalGraphicsInc/cesium/builds/319382855?utm_source=github_status&utm_medium=notification). Can you resolve those?

File: Source/Core/Credit.js [hunk 1]
```diff
@@ -30,19 +30,13 @@ define([
      * //Create a credit with a tooltip, image and link
      * var credit = new Cesium.Credit('Cesium', '/images/cesium_logo.png', 'http://cesiumjs.org/');
      */
-    function Credit(options, imageUrl, link) {
+    function Credit(options) {
         var text;
         var showOnScreen;
```
Comment (ggetz) [inline Source/Core/Credit.js]: @Jahrkor We can now declare these variables when they are assigned below. Other than that this all looks good!

Comment (Jahrkor) [reply Source/Core/Credit.js]: Thanks @ggetz

**Comment by hpinkos** [comment]
Great @Jahrkor !  One last thing:  can you remove the `deprecationWarning` from the requires at the top of the file?

**Comment by hpinkos** [comment]
Thanks for the pull request @Jahrkor!  Since this had to make it in for the release today, we finished this change in another branch.  I'm going to close this pull request.

If you're interested in contributing something else, check out the beginner label: https://github.com/AnalyticalGraphicsInc/cesium/issues?q=is%3Aissue+is%3Aopen+label%3Abeginner

Thanks again!

**Comment by pjcozzi** [comment]
Sorry @Jahrkor, I missed this and opened #6075 in a hurry for the release today.  Future contributions are very much welcome!

**Pull request closed without merging**